packages/p2p-v2/src/components/Input/Input.scss

```
@@ -0,0 +1,58 @@
+.p2p-v2-input {
+    display: flex;
+    justify-content: center;
+    margin: 0.5rem 0 3.6rem;
+    width: 100%;
+
+    @include mobile {
+        flex-direction: column;
+        margin-bottom: 1rem;
+    }
+
+    &__error {
+        padding-top: 0.2rem;
+        position: absolute;
+        margin-left: 1rem;
+        bottom: 13.3rem;
+        left: 4.5rem;
+
+        @include mobile {
+            bottom: 0;
+            box-decoration-break: clone;
+            padding-top: 0.4rem;
+            margin: 0 0 0.6rem 1.6rem;
+            margin-bottom: 0.6rem;
+            position: relative;
+            left: 0;
+        }
+    }
+
+    &__field {
+        display: flex;
+        width: 92%;
+        height: 4rem;
+        padding: 1rem 1.6rem;
+        justify-content: center;
+        align-items: center;
+        border-radius: 4px;
+        border: 1px solid #d6dadb;
+
+        @include mobile {
+            width: 100%;
+        }
+
+        &:focus-visible {
+            outline: none;
+            border-color: #85acb0;
+        }
+
+        &::placeholder {
+            color: #999;
+        }
+
+        &--error {
+            // stylelint-disable-next-line declaration-no-important
+            border-color: #ec3f3f !important;
```

```suggestion
            border-color: #ec3f3f !important;
```
@ameerul-deriv, isn't there a way to increase the specificity of this selector, as opposed to disabling the warning? 🤔

---

hmmm i couldn't because on border focus, we need that, but if theres an error, I need to override it, but I think this is fine for now, since Aizad is done with the input and we can replace this 🙏